src/test/java/com/ibm/stocator/fs/swift2d/TestSwiftFileSystemBlockLocation.java

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
@djalova thanks for the comments. @jasoncl can you provide another patch addressing the comments?
I also not aure about testing block locations as a stand alone test, seems it should be moved to some new test that will test configuration only.